cronie_common.c

```diff
@@ -107,6 +107,9 @@ int expand_envvar(const char *source, char *result, size_t max_size) {
         }
 
         envvar_name = malloc(envvar_name_size + 1);
+	if (envvar_name == NULL)
+		goto too_big;
+
```

Comment:
One more nit - could you please replace tabs with spaces? Also the goto should be then aligned with the condition (i.e., 4 spaces indentation per level)

Reply:
I resubmitted the patch.